### PR TITLE
Fix usage of nil values

### PR DESCRIPTION
**Pull request by BackrndSource**
Im having this issue:

```
Error running call reference function for resource es_extended: citizen:/scripting/lua/scheduler.lua:403: @es_extended/common/functions.lua:37: bad argument #1 to 'upper' (string expected, got nil)
stack traceback:
        [C]: in function 'string.upper'
        @es_extended/common/functions.lua:37: in field 'GetWeaponLabel'
        @es_extended/server/classes/player.lua:385: in upvalue 'ref'
        citizen:/scripting/lua/scheduler.lua:389: in function <citizen:/scripting/lua/scheduler.lua:388>
        [C]: in function 'xpcall'
        citizen:/scripting/lua/scheduler.lua:388: in function <citizen:/scripting/lua/scheduler.lua:387>
stack traceback:
        [C]: in function 'error'
        citizen:/scripting/lua/scheduler.lua:403: in function <citizen:/scripting/lua/scheduler.lua:372>
```

This will fix it.

### CODE REVIEW

**Comment by BackrndSource** [comment]
Lasto corrections because i'm having this issue too:
```
Error running system event handling function for resource es_extended: citizen:/scripting/lua/scheduler.lua:41: Failed to execute thread: @es_extended/server/main.lua:277: attempt to index a nil value (local 'xPlayer')
stack traceback:
        @es_extended/server/main.lua:277: in upvalue 'handler'
        citizen:/scripting/lua/scheduler.lua:219: in function <citizen:/scripting/lua/scheduler.lua:218>
stack traceback:
        [C]: in function 'error'
        citizen:/scripting/lua/scheduler.lua:41: in field 'CreateThreadNow'
        citizen:/scripting/lua/scheduler.lua:218: in function <citizen:/scripting/lua/scheduler.lua:182>

```

**Comment by Mallogado** [comment]
solution?

**Comment by Mallogado** [comment]
I always have this problem and I do not know how to solve it

citizen:/scripting/lua/scheduler.lua:41:
citizen:/scripting/lua/scheduler.lua:218

File: server/main.lua [hunk 1]
```diff
@@ -371,16 +377,19 @@ AddEventHandler('esx:removeInventoryItem', function(type, itemName, itemCount)
 			TriggerClientEvent('esx:showNotification', _source, _U('imp_invalid_amount'))
 		else
 			local xPlayer = ESX.GetPlayerFromId(source)
-			local playerCash = xPlayer.getMoney()
+				
+			if xPlayer ~= nil then
+				local playerCash = xPlayer.getMoney()
```
Comment (dilaraagat) [inline server/main.lua]: forgot an end?

**Pull request closed without merging**